src/main/webapp/assets/svg/linkedin.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="15" height="15" viewBox="0 0 15 15" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M7.5 0C3.35813 0 0 3.35813 0 7.5C0 11.6419 3.35813 15 7.5 15C11.6419 15 15 11.6419 15 7.5C15 3.35813 11.6419 0 7.5 0ZM6.25 10H5V6.25H6.25V10ZM5.625 5.69313C5.24563 5.69313 4.9375 5.38312 4.9375 5C4.9375 4.6175 5.245 4.30687 5.625 4.30687C6.005 4.30687 6.3125 4.6175 6.3125 5C6.3125 5.38312 6.00437 5.69313 5.625 5.69313ZM10.625 10H9.37625V8.21187C9.37625 7.03625 8.125 7.13562 8.125 8.21187V10H6.875V6.25H8.125V6.93313C8.67 5.92313 10.625 5.84812 10.625 7.90062V10Z" fill="black"/>
+</svg>
```

Comment:
What do you think about using a LinkedIn logo that has a larger `in` wordmark? It's pretty small next to the GitHub logo and might be difficult to read :0

Reply:
Good idea! Adding right now!